packages/paste-website/src/pages/components/toggle/index.mdx

```diff
@@ -0,0 +1,144 @@
+import {basicToggle, onToggle, disabledToggle} from '../../../component-examples/ToggleExamples.ts';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/toggle"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/toggle/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Toggle"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/toggle"
+  storybookUrl="/?path=/story/components-toggle--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Toggle
+
+A Toggle is an interactive binary switch. It should be used in forms when an "on/off" or "yes/no" input is needed. Use label text to describe the purpose of the Toggle. The label text shouldn't change when the state of the Toggle changes. Help Text can also be used to provide additional guidance.
+
+### Accessibility
+
+Toggle has `role="switch"`. The `aria-labelledby` attribute is automatically set on the component, which ensures that assisstive technology devices can read the Toggle's label. When the Toggle is on, `aria-selected` is set to `"true"` (it defaults to `"false"`).
+
+## Examples
+
+### Default
+
+<LivePreview scope={{Toggle, useUID}}>{basicToggle}</LivePreview>
+
+### Disabled
+
+<LivePreview scope={{Toggle, useUID}}>{disabledToggle}</LivePreview>
+
+## States
+
+### On
+
+The default Toggle state is "off"
+
+<LivePreview scope={{Toggle, useUID}}>{onToggle}</LivePreview>
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/toggle - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {Toggle} from '@twilio-paste/core/toggle';
+
+export const Basic = () => {
+  return <Toggle>Label</Toggle>;
+};
+```
+
+#### Props
+
+| Prop        | Type                           | Description                                                                               | Default    |
+| ----------- | ------------------------------ | ----------------------------------------------------------------------------------------- | ---------- |
+| `children`  | `NonNullable<React.ReactNode>` | Pass the label text as children                                                           |            |
```

Comment:
suggestion: this is more an instruction than description. I think this belongs in the default example and that this should just say "Text that is used as the label"